### PR TITLE
feat(cast): ux upgrade cast block returning block time in readable format. 

### LINKED ISSUE #8183: feature request for cast(UX upgrade): cast block returning block time in human-readable string.
### Component

Cast

### Describe the feature you would like

Every time I query cast block 
```shell
cast block 15944892 --rpc-url=https://rpc.ankr.com/base
```

The response would be like: 
```shell
baseFeePerGas        12951667
difficulty           0
extraData            0x
...
timestamp            1718679131
...
```

I have to copy the timestamp in unix and parse somewhere convert to human-readable GMT string. 
like following:
```shell
baseFeePerGas        12951667
difficulty           0
extraData            0x
gasLimit             90000000
...
timestamp            1718679131	GMT 2024-06-18 02:52:11
```

Or: 

```shell 
baseFeePerGas        12951667
difficulty           0
extraData            0x
gasLimit             90000000
...
timestamp            1718679131
time                 GMT 2024-06-18 02:52:11
```

I want to make a PR for it. But no sure if this would be accepted. 

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by JamieShip**
## Motivation

support readable time for `cast block` 

Closes #8183 

## Solution

Add extra column `time` in the response

```shell
cast block 15954127 --rpc-url=https://rpc.ankr.com/base
```

example response:

```shell
baseFeePerGas        2364687
difficulty           0
extraData            0x
gasLimit             90000000
gasUsed              6998016
...
timestamp            1718697601
time                 Tue, 18 Jun 2024 08:00:01 +0000
...
```

### CODE REVIEW

File: crates/common/src/fmt/ui.rs [hunk 1]
```diff
@@ -489,6 +489,7 @@ sha3Uncles           {}
 size                 {}
 stateRoot            {}
 timestamp            {}
+time                 {}
```
Comment (DaniPopes) [inline crates/common/src/fmt/ui.rs:492]: maybe a bit misleading to have its own row, I would put it after timestamp in parentheses

**Review by DaniPopes — CHANGES_REQUESTED** [review]
Thanks! One nit

File: crates/common/src/fmt/ui.rs (hunk 1 above)
Comment (JamieShip) [reply crates/common/src/fmt/ui.rs:492]: I would change it. 
Is it ok to be like
```
timestamp            {}\t{}
```
Or any delimiter recommended?

**Comment by JamieShip** [comment]
resolved as you requested. Not sure if the delimiter '\t' is the best option.

**Comment by JamieShip** [comment]
> Thanks! One nit

resolved as you requested. Not sure if the delimiter '\t' is the best option.

**Review by DaniPopes — APPROVED** [review]

**Pull request merged**